src/consumers.ts

```diff
@@ -109,51 +133,69 @@ class EmptyResultEngine extends AnalysisConsumer implements DiagnosticProducer
     }
 
     produce(): Diagnostic[] {
-        if (this.item == {} || 
-            this.item.finished_at === undefined ||
-            this.item.finished_at == null) {
+        if (this.item == {} && (this.item.finished_at === undefined ||
+            this.item.finished_at == null)) {
             return [{
                 severity: DiagnosticSeverity.Information,
                 range: get_range(this.context.version),
                 message: `Application dependency ${this.context.name.value}-${this.context.version.value} - analysis is pending`,
-                source: 'Dependency Analytics'
+                source: 'Dependency Analytics '
             }]
         } else {
             return [];
         }
     }   
 }
 
+let targer_link : DocumentUri;
```

Comment:
What is the use of this global var?

Reply:
This is a DocumentUri type, to pass the Uri object as a string and `this.regLink` is type of string and it should be converted to a Uri object.